roles/firewall/commit/tasks/main.yml

```diff
@@ -1,20 +1,24 @@
 ---
 - name: Enable service ports via iptables
+  delegate_to: "{{ item.delegate|default(inventory_hostname) }}"
```

Comment:
@paramite I wanted to highlightthis change, because `default(omit)` wasn't useful here; it ended up with (I think) an empty `delegate_to`, which caused the connection to fail.  I think defaulting instead to `inventory_hostname` should be correct.